packages/sdk/src/modules/passkey/types.ts

```diff
@@ -0,0 +1,5 @@
+export enum Batch32Prefix {
+  PASSKEY = 'passkey',
+}
+
+export type Batch32 = `${Batch32Prefix}.${string}`;
```

Comment:
```suggestion
export enum BechPrefix {
  PASSKEY = 'passkey',
}

export type Bech32 = `${Batch32Prefix}.${string}`;
```
just a typo fix _Batch_ -> **Bech**